src/server/rest/v1/service/SiteService.ts

```diff
@@ -600,7 +601,7 @@ export default class SiteService {
     // Check data is valid
     UtilsService.checkIfSiteValid(filteredRequest, req);
     // Check static auth for reading company
-    if (!Authorizations.canReadCompany(req.user)) {
+    if (!(await Authorizations.canReadCompany(req.user)).authorized) {
```

Comment:
Is `.authorized` really needed here?

Reply:
This code is no longer here after merge with master-qa :-)